packages/snap-preact-components/src/components/Molecules/Checkbox/Checkbox.tsx

```diff
@@ -13,8 +13,7 @@ import { Icon, IconProps } from '../../Atoms/Icon';
 import { useA11y } from '../../../hooks/useA11y';
 
 const CSS = {
-	checkbox: ({ size, color, theme }: Partial<CheckboxProps>) => {
-		const pixelSize = isNaN(Number(size)) ? size : `${size}px`;
+	checkbox: ({ pixelSize, color, theme }: Partial<CheckboxProps>) => {
```

Comment:
`pixelSize` is not a CheckboxProp... which may cause issues when moving this to templates where these types are more strict... 🤔